### DISCUSSION TITLE
How does the compiler handle unknown `__declspec`?

### DISCUSSION THREAD

**Discussion by ghost** [LADSoft/OrangeC#1014] [General]
It seems only `__declspec(dllexport)` and `__declspec(dllimport)` are supported. MSVC has other `__declspec`, too. For example: `__declspec(allocator)`, `__declspec(noalias)`, `__declspec(restrict)`,... Sometimes, people forget to encapsulate these `__declspec` with `#ifdef _MSC_VER`. This will cause problem when using non-MSVC compilers. Please let me know how Orange C handles the unknown `__declspec`. Thank you.

**Comment by LADSoft** [comment]
apparently, it at least prints a message here that the declspec() is being ignored.    Might help if it further clarified what it is ignoring though...
